src/cockpit/internal_endpoints.py

```diff
@@ -76,10 +76,6 @@ def update(self, *args):
         ...
 
 
-class cockpit_Packages(bus.Object):
-    ...
-
-
```

Comment:
This makes me happy.  I think as we implement more and more of this functionality "for real" we might notice that this file disappears...